src/graph.jl

```diff
@@ -96,3 +96,14 @@ function get_attributes(raw_graph::Dict{String, Any},
     end
     return attributes
 end
+
+function get_population_of_subgraph(graph::BaseGraph, nodes::Set{Int})::Int
+    """ Returns the population of the subgraph induced by `nodes`.
+        Potential TODO: `nodes` is a set, and iterating over a set is slower than an array.
```

Comment:
is it? (idk)

Reply:
Sets are hash tables in disguise (I would think they are implemented as a hash table that can only have one value per key to maintain uniqueness, with a really strong unique hash generator like uuid but im speculating), and are optimized for operations like `if item in set` checks which would be O(1) vs `if item in list` which would be O(n). Lists are contiguous blocks of memory precisely favorable for iteration.

Regardless when I first wrote my implementation I remember profiling this and saw some performance improvement over large runs. I put it only as a `Potential TODO` because I don't feel too strongly about this (unless later profiling flags this.)